src/templates/static/affiliates/signup.jsx

```diff
@@ -13,13 +13,13 @@ const Signup = () => (
         <div className='container'>
             <h1>{it.L('[_1] Affiliate Programme', it.website_name)}</h1>
             <p>
-                {it.L('Earn up to 35% commission with an award-winning binary options trading platform.')}
+                {it.L('Join the Binary.com Affiliate Programme and earn commission by referring new clients to our award-winning binary options trading platform.')}
```

Comment:
instead of `Binary.com` please use a placeholder `[_1]` with replacement as `it.website_name`